integration/operator_test.go

```diff
@@ -252,27 +252,31 @@ func operatorSetup() error {
 		return microerror.Mask(err)
 	}
 
-	if err := writeAWSResourceValues(); err != nil {
-		return microerror.Maskf(err, "unexpected error writing aws-resource-lab values file")
+	err := writeAWSResourceValues()
+	if err != nil {
+		return microerror.Maskf(err, "writing aws-resource-lab values file")
 	}
 
-	if err := runCmd("helm registry install quay.io/giantswarm/aws-resource-lab-chart:stable -- -n aws-resource-lab --values " + awsOperatorValuesFile); err != nil {
-		return microerror.Maskf(err, "unexpected error installing aws-resource-lab chart: %v")
```

Comment:
There are formatting placeholders used without providing values to interpolate them with the actual message. The output was weird in some cases.